8.8/jhbuildrc

```diff
@@ -130,6 +126,10 @@ makeargs = '-j1'
 
 use_ninja = False
 
-# we want to force in-tree builds ... some packages (like matio) break with 
-# out of tree builds
-buildroot = None
+# we want to force out of tree builds ... some packages (like harfbuzz)
+# break with in-tree builds
+buildroot = os.path.join(basedir, 'build')
```

Comment:
Changed, otherwise Harfbuzz fails:
https://github.com/harfbuzz/harfbuzz/blob/14e1fabc41a9a5ead3d91d560773050469982f54/CMakeLists.txt#L20

Matio seems now also to work with out of tree builds.